src/OpenTelemetry.AutoInstrumentation/Configurations/EnvironmentConfigurationMetricHelper.cs

```diff
@@ -29,7 +29,12 @@ public static MeterProviderBuilder UseEnvironmentVariables(
                 MetricInstrumentation.HttpClient => Wrappers.AddHttpClientInstrumentation(builder, lazyInstrumentationLoader),
                 MetricInstrumentation.NetRuntime => Wrappers.AddRuntimeInstrumentation(builder, pluginManager),
                 MetricInstrumentation.Process => Wrappers.AddProcessInstrumentation(builder),
-                MetricInstrumentation.NServiceBus => builder.AddMeter("NServiceBus.Core"),
+                MetricInstrumentation.NServiceBus => builder
+#if NET6_0_OR_GREATER
+                    .AddMeter("NServiceBus.Core.Pipeline.Incoming") // NServiceBus 9.1.0+
+#endif
+                    .AddMeter("NServiceBus.Core"), // NServiceBus [8,0.0, 9.1.0)
+
```

Comment:
Wouldn't it be more consistent to use NET8_0_OR_GREATER, as used elsewhere in the code?

Reply:
We are targeting this file only for net462 (all .net FX versions) and net6.0 (all .Net versions). It is the reason I put conditional compilation for net6.